org.eclipse.xtext.ui.tests/tests/org/eclipse/xtext/ui/tests/editor/hover/AbstractProblemHoverTest.java

```diff
@@ -98,6 +155,11 @@ protected IRegion getHoverRegionInternal(int lineNumber, int offset) {
 		protected Object getHoverInfoInternal(ITextViewer textViewer, int lineNumber, int offset) {
 			return null;
 		}
+		
+		@Override
```

Comment:
is this still needed?

Reply:
you could add a comment that it makes the getter visible to the test and thus is needed